extensions/amp-social-share/0.1/amp-social-share.js

```diff
@@ -92,9 +92,14 @@ class AmpSocialShare extends AMP.BaseElement {
       // mailto:, whatsapp: protocols breaks when opened in _blank on iOS Safari
       const isMailTo = /^mailto:$/.test(parseUrl(href).protocol);
       const isWhatsApp = /^whatsapp:$/.test(parseUrl(href).protocol);
+      const isSMS = /^sms:$/.test(parseUrl(href).protocol);
```

Comment:
What are you suggesting I change? I just followed the conventions that already existed. It's probably best as a variable as I had to use it twice.